mscore/events.cpp

```diff
@@ -330,6 +331,7 @@ void ScoreView::mousePressEventNormal(QMouseEvent* ev)
                         r.translate(0.0, -m->spatium());
                         }
                   if (r.contains(editData.startMove)) {
+                        //TourHandler::startTour("select-tour");
```

Comment:
Reason for this?

Reply:
Well, I am waffling on whether it's a good idea, I left it in so we can easily add it back.  For now, I'm erring on side of not beating people over the head with too many tours too soon.